### PR TITLE
Add link to NABS as another similar solution

### PR DESCRIPTION
**Pull request by StevenACoffman**
Might be possible to have them combine efforts here.

**What**: Add readme link to [NABS](https://github.com/artlogic/nabs) other solution

**Why**: NABS (not another build system) is very similar to nps

**How**: typing. ;P

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -415,6 +415,7 @@ called [`npmsearch-cli`](https://www.npmjs.com/package/npmsearch-cli).
 - [scripty][scripty] has a solution for this problem as well. The reason I didn't go with that though is you still need
 a line for every script (one of the pains I'm trying to solve) and a each script requires its own file (one of the
 benefits of npm scripts I wanted to keep).
+- [nabs][nabs] is a compiler that turns a nicely structured YAML file into script entries in your package.json
```
Comment (kentcdodds) [inline README.md]: For this to be linked properly, you'll want to add a `nabs: <URL here>`  entry at the bottom with the rest of the URLs :)

**Comment by kentcdodds** [comment]
> Might be possible to have them combine efforts here.

They seem like fairly significantly different projects. But I'm happy to include a link.

File: README.md (hunk 1 above)
Comment (StevenACoffman) [inline README.md]: Ah! I wondered how that magic worked. I thought it was just that I wasn't viewing it in the proper context. Thanks for your patience, and for making a cool thing!

**Comment by StevenACoffman** [comment]
Fixed. I didn't add myself as a contributor, as I don't feel like I deserve any credit for any substantial improvement. This link is sort of silly. I only added it because I kept mixing the projects up.

**Pull request merged**